jsplot/src/main/java/tech/tablesaw/plotly/Utils.java

```diff
@@ -53,4 +56,53 @@ public static String dataAsString(double[][] data) {
     builder.append("]");
     return builder.toString();
   }
+
+  /**
+   * Returns a list of column without missing value
```

Comment:
It looks to me like you didn't make this change. Can you explain why? Your solution requires almost 40 lines of code. Mine would require about five i think.

Reply:
I personally don't prefer this method. I think making a copy of a large table is not efficient. Sometimes you may need to convert a double or object array to a column and create tables.

Reply:
If you want to discuss your preferences, you should have explained them earlier instead of ignoring my comment. You're wasting my time and that is not efficient for me.